packages/starknet-snap/test/src/getErc20TokenBalance.test.ts

```diff
@@ -51,7 +51,10 @@ describe('Test function: getErc20TokenBalance', function () {
     };
     apiParams.requestParams = requestObject;
     const result = await getErc20TokenBalance(apiParams);
-    expect(result).to.be.eq('0x64a');
+    expect(result).to.be.eql({
+      spendableBalance: hexAmount,
+      totalBalance: hexAmount,
+    });
   });
 
   it('should get ERC-20 token balance with BlockIdentifier pending if the account is deployed', async function () {
```

Comment:
test case has to change, as we are not return balance by account is deployed, but the smaller one